src/app/Providers/components/ProvidersTable/VMware/VMwareProviderHostsTable/VMwareProviderHostsTable.tsx

```diff
@@ -35,7 +36,9 @@ const VMwareProviderHostsTable: React.FunctionComponent<IVMwareProviderHostsTabl
   };
   const { sortBy, onSort, sortedItems } = useSortState(hosts, getSortValues);
 
-  const { selectedItems, toggleItemSelected, selectAll } = useSelectionState<IHost>(sortedItems);
+  const { selectedItems, toggleItemSelected, selectAll } = useSelectionState<IHost>({
+    items: sortedItems,
```

Comment:
To be safe, let's get in the habit of also passing an `isEqual` param here. I think all hosts will have a `metadata.uid` property, which we should add to our `IHost` interface and our mock data. Then we can use:
```
isEqual: (a, b) => a.metadata.uid === b.metadata.uid
```

And then below when we define the rows, we can use `isItemSelected(host)` from the hook in place of `selectedItems.includes(host)`.

This way if the hosts array changes in redux due to polling, we won't lose our selection state (since `includes` depends on the redux objects being referentially equal (`===`) to the ones that were present when the user selected them.

Honestly, this may be enough of a gotcha that we might want to make `isEqual` a required param. I could see this becoming a sneaky bug in places we neglect it.